apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/(marketplace)/components/list-button.tsx

```diff
@@ -40,14 +46,35 @@ export const CreateListingButton: React.FC<CreateListingButtonProps> = ({
         </SheetTrigger>
         <SheetContent className="w-full overflow-y-auto sm:min-w-[540px] lg:min-w-[700px]">
           <SheetHeader>
-            <SheetTitle className="mb-5 text-left">{createText}</SheetTitle>
+            <SheetTitle className="text-left">{createText}</SheetTitle>
           </SheetHeader>
-          <CreateListingsForm
-            contract={contract}
-            type={type}
-            actionText={createText}
-            setOpen={setOpen}
+          <TabButtons
+            tabs={LISTING_MODES.map((mode) => ({
+              name: mode,
+              isActive: mode === listingMode,
+              onClick: () => setListingMode(mode),
+              isEnabled: true,
+            }))}
+            tabClassName="text-sm gap-2 !text-sm"
+            tabContainerClassName="px-3 pt-1.5 gap-0.5"
           />
+          <div className="mt-5 px-3">
+            {listingMode === "Selection" ? (
+              <CreateListingsForm
```

Comment:
Lets remove the `px-3 pt-1.5`  in `tabContainerClassName` and instead add margin bottom on the sheet title "Create Direct Listing" 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/2361b67b-f520-43c5-9ae2-1863427f1706.png)

Reply:
updated